Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -1490,7 +1490,7 @@ void Document::obtain_theme_color()
             auto media = element.attribute(HTML::AttributeNames::media);
             if (media.has_value()) {
                 auto query = parse_media_query(context, media.value());
-                if (window() && !query->evaluate(*window()))
+                if (query.is_null() || (window() && !query->evaluate(*window())))
```

Comment:
Should we be continuing here if `window()` is null? I would have thought we'd want something like: `if (!(query && window() && query->evaluate(*window())))` here.

Reply:
That would make more sense, I kept the `window && !evaluate` because that's what the initial implementation did.